### PR TITLE
[Feature] Support ConvNeXt 

### PR DESCRIPTION
**Pull request by hhaAndroid**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Support ConvNeXt.  

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by ZwwWayne** [comment]
Need to resolve conflicts.

File: configs/convnext/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+[A ConvNet for the 2020s](https://arxiv.org/abs/2201.03545)
+
+## Introduction
+
+<!-- [BACKBONE] -->
+
+<a href="https://github.com/facebookresearch/ConvNeXt">Official Repo</a>
+
+<a href="https://github.com/open-mmlab/mmclassification/blob/v0.22.0/mmcls/models/backbones/convnext.py#L133">Code Snippet</a>
```
Comment (jbwang1997) [inline configs/convnext/README.md:11]: Need to adapt README.md to the new format.

File: configs/convnext/metafile.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        Metrics:
+          mask AP:
+    Weights:
+    Paper:
+      URL: https://arxiv.org/abs/2201.03545
+      Title: 'A ConvNet for the 2020s'
+    README: configs/convnext/README.md
+    Code:
+      URL: https://github.com/open-mmlab/mmdetection/blob/v2.16.0/mmdet/models/backbones/swin.py#L465
+      Version: v2.24.0
```
Comment (jbwang1997) [inline configs/convnext/metafile.yml:30]: Remember to change the version.

**Comment by jbwang1997** [comment]
Other parts look good to me.

**Comment by hhaAndroid** [comment]
> onvnext as backbone!

If you just want the backbone, you can refer to the configuration and use the mmcls directly. like this: 

```python
backbone=dict(
        _delete_=True,
        type='mmcls.ConvNeXt',
        arch='tiny',
        out_indices=[0, 1, 2, 3],
        drop_path_rate=0.4,
        layer_scale_init_value=1.0,
        gap_before_final_norm=False,
        init_cfg=dict(
            type='Pretrained', checkpoint=checkpoint_file,
            prefix='backbone.')),
```

**Comment by ZwwWayne** [comment]
Is there any progress?

**Comment by MiXaiLL76** [comment]
any update?

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**